src/command.rs

```diff
@@ -118,6 +118,10 @@ pub fn run() -> sc_cli::Result<()> {
             let runner = cli.create_runner(cmd)?;
             runner.sync_run(|config| cmd.run(config.chain_spec, config.network))
         }
+        Some(Subcommand::PurgeAbciStorage(ref cmd)) => {
```

Comment:
We do not need to have to different cmd commands PurgeAbciStorage and PurgeChain. It should be only one cmd command PurgeChain which will remove all storages: original and our storage.